critiquebrainz/frontend/release/views.py

```diff
@@ -0,0 +1,17 @@
+from flask import Blueprint, redirect
+from critiquebrainz.frontend.apis import musicbrainz
+from werkzeug.exceptions import NotFound
+
+release_bp = Blueprint('release', __name__)
+
+
+@release_bp.route('/<uuid:id>')
+def entity(id):
+    id = str(id)
+    if musicbrainz.get_release_by_id(id):
+        release_data = musicbrainz.get_release_by_id(id)
```

Comment:
Instead of making two expensive queries, you can do only one. Please combine these.